src/components/TodoContent/TodoFooter.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, useContext } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../TodoContext';
+import { FilterOptions } from '../../types/FillterOptions';
+
+export const TodoFooter: FC = () => {
+  const {
+    todosCount,
+    activeTodosLeft,
+    filterBy,
+    onFilterByChange,
+  } = useContext(TodoContext);
+
+  const filterLinkClasses = (currentOption: FilterOptions) => {
+    return classNames('filter__link', {
+      selected: filterBy === currentOption,
+    });
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {activeTodosLeft}
+        {' '}
+        items left
```

Comment:
Add right condition for `1 item left`
```suggestion
        {`${activeTodosLeft} items left`}
```